src/content/guides/requesting-pre-auth.mdx

```diff
@@ -19,65 +19,44 @@ Pre Auth payments are not supported in all cases.
 
 Pre Auth payments go through an orthogonal payment flow compared to Centrapay’s standard payment flow.
 
+> The example below shows the integrator polling for the Payment Request, this is not needed for eCommerce flows as the integrator will be informed when the authorization has completed.
+
```

Comment:
I feel like this covers off the confusion eCommerce integrator's might have. We could add alternate sequence diagram but I'm not sure it's needed at this point.